esp-hal/src/prelude.rs

```diff
@@ -38,4 +38,4 @@ pub use crate::timer::timg::{
 pub use crate::timer::Timer as _esp_hal_timer_Timer;
 #[cfg(any(uart0, uart1, uart2))]
 pub use crate::uart::Instance as _esp_hal_uart_Instance;
-pub use crate::{entry, macros::*, InterruptConfigurable};
+pub use crate::{clock::CpuClock, entry, macros::*, Config, InterruptConfigurable};
```

Comment:
I'm not sure we should include `Config` in the prelude, I know it makes sense to, but I fear there will be _many_ import conflicts with such a generic name (using anything from embassy, like USB or embassy-net will run into this issue).

Imo this might be one of the rare cases where we just qualify the full path in its usage? i.e `esp_hal::init(esp_hal::Config::default())` doesn't seem too bad.

`CpuClock` is fine though.

Reply:
`esp_hal::init(Default::default())` would also work